backend/support/test_cog_over.py

```diff
@@ -342,6 +342,7 @@ def test_cog_assignment_with_uei_in_baseline(self):
         sac.general_information["auditee_uei"] = UEI_WITH_BASELINE
         baker.make(
             CognizantBaseline,
+            ein=sac.general_information["ein"],
```

Comment:
The intent here was to test the case where there is a matching UEI in the baseline regardless of the value of the ein. Adding an ein makes it the same as a different test case.

Reply:
Test case is updated handle 'uei only' case.